### DISCUSSION TITLE
当前页面只对Apollo管理员开放

### DISCUSSION THREAD

**Discussion by fyzzz** [apolloconfig/apollo#4728] [Q&A]
版本：java-2.0.1
登录用户：apollo
打开“开放平台授权管理”页面，首先显示“当前页面只对Apollo管理员开放”，然后跳转到创建第三方应用页面，如下图：
<img width="813" alt="image" src="https://user-images.githubusercontent.com/37723646/218018755-8ee3862d-aa1b-46d6-84a4-e14fb9c7a4f0.png">
同样的问题还出现在“用户管理”页面

**Comment by nobodyiam** [comment]
What's the actual issue?
BTW, you may upgrade to [v2.1.0](https://github.com/apolloconfig/apollo/releases/tag/v2.1.0) which provides a better UI experience.

**Reply by fyzzz** [reply]
你好，我通过quickstart部署使用，根据官网描述，/open/manage.html（即"开放平台授权管理"）应展示第三方应用列表，但我打开后提示“当前页面只对Apollo管理员开放”，然后跳转到创建第三方应用页面，但是我已经是以apollo用户登录了，同样的问题还出现在“用户管理“页面。

**Reply by nobodyiam** [reply]
That's because the quick start was 2.0.1 version and I have just [updated](https://github.com/apolloconfig/apollo-quick-start/commit/4a71cc90ea8fe2d1f0aede5ebac1b2f115cef3cb) it to 2.1.0 version. You may fetch the latest version and try again.
